pwa/src/components/PatientInfoForm.tsx

```diff
@@ -0,0 +1,265 @@
+import React, { ReactElement, useState } from "react";
+import {
+  InputItem,
+  List,
+  TextareaItem,
+  Button,
+  Radio,
+  WhiteSpace,
+} from "antd-mobile";
+import "../scss/PatientInfoForm.scss";
+
+const RadioItem = Radio.RadioItem;
+
+export default function Form(): ReactElement {
+  // State Hooks for Radio Buttons
+  // useState returns a value, and a function to set that value
+  const [sex, setSex] = useState(null as Number); // Default: null
+  const [albendazole, setAlbendazole] = useState(0); // Default: No
+  const [visit, setVisit] = useState(0); // Default: No
+  const [pregnant, setPregnant] = useState(0); // Default: No
+  const [planning, setPlanning] = useState(0); // Default: No
+
+  // Data for Radio Buttons
+  const sexValues = [
+    { value: 0, label: "Male" },
+    { value: 1, label: "Female" },
+  ];
+
+  const radioYesNo = [
+    { value: 0, label: "No" },
+    { value: 1, label: "Yes" },
+  ];
+
+  /* istanbul ignore next */
+  const onSubmit = () => {
+    // This shows how to access the values of the form
+    const firstNameField: HTMLInputElement = document.getElementById(
+      "firstname"
+    ) as HTMLInputElement;
+    const lastNameField: HTMLInputElement = document.getElementById(
+      "lastname"
+    ) as HTMLInputElement;
+    const nicknameField: HTMLInputElement = document.getElementById(
+      "nickname"
+    ) as HTMLInputElement;
+    const birthdateField: HTMLInputElement = document.getElementById(
+      "birthdate"
+    ) as HTMLInputElement;
+    const complaintField1: HTMLInputElement = document.getElementById(
+      "complaint1"
+    ) as HTMLInputElement;
+    const complaintField2: HTMLInputElement = document.getElementById(
+      "complaint2"
+    ) as HTMLInputElement;
+    const complaintField3: HTMLInputElement = document.getElementById(
+      "complaint3"
+    ) as HTMLInputElement;
+
+    console.log("First name:", firstNameField.value);
+    console.log("Last name:", lastNameField.value);
+    console.log("Nickname:", nicknameField.value);
+    console.log("Birth Date:", birthdateField.value);
+    console.log(
+      "Complaints: ",
+      complaintField1.value +
+        " " +
+        complaintField2.value +
+        " " +
+        complaintField3.value
+    );
+    console.log("Wants Planning?: " + planning);
+  };
```

Comment:
console log commands should be removed before merging into dev unless they are blocked off by some global `DEBUG` variable.

Reply:
Added this, [here's a link to the commit](https://github.com/Haiti-Distributed-Healthcare-System/hdhs/pull/66/commits/f89f9adcb19bc3273fa8ff6af1b9327d2043b7bf).

Logs will now only show if NODE_ENV is "development". This is the default when running `npm start` [due to create react app](https://create-react-app.dev/docs/adding-custom-environment-variables/). Note that within create react app this cannot be overridden. Building the front end and serving the output 

![image](https://user-images.githubusercontent.com/33632286/79670854-7913cf00-818b-11ea-9fbf-d305f4f4d184.png)

Reply:
The purpose of the console logs was to have a documented way of accessing the form fields, until they are used appropriately for data submission. Would you recommend removing this completely or would it be fine for the block to be commented out?

Reply:
I think after adding the development conditional, this can be resolved right @abpwrs ? you just meant we should keep the console log clean for any users at all times.

Reply:
Correct, this can be resolved. I think it is helpful for debugging purposes. It just doesn't need to be visible in the production application.